### ISSUE TITLE
Have fun and learn deeply with a coding study

### ISSUE THREAD

**Issue by DavidMatalik** [bouwe77/curious-coders#29]

**Comment by bouwe77** [comment]
Sounds interesting, can you elaborate?

**Comment by RichStone** [comment]
coders are artists in a way

and you can practice coding like artists practice art to hone your craft

it's quick and fun

https://www.youtube.com/watch?v=33fAzjOTaDE

**Comment by RichStone** [comment]
upvote ⬆️

**Comment by bouwe77** [comment]
Yeah I like ( and apply) the idea of coding (simulating) real-life things, so much more fun than the next to do or CRUD app...

**Comment by RichStone** [comment]
> Yeah I like ( and apply) the idea of coding (simulating) real-life things, so much more fun than the next to do or CRUD app...

nice! Do you have a recent example?

**Comment by bouwe77** [comment]
I have less than I thought, but the coffee machine over here is one: https://bouwe.io/videos

**Comment by RichStone** [comment]
nice groovy musix in there :))
